doc/users/next_whats_new/title_will_not_overlap_xaxis.rst

```diff
@@ -0,0 +1,8 @@
+Axes title will no longer overlap xaxis
+---------------------------------------
+
+Previously the axes title had to be moved manually if an xaxis overlapped
+(usually when the xaxis was put on the top of the axes).  The title can
+still be placed manually.  However, titles that need to be moved are
+at ``y=1.0``,  so manualy placing at 1.0 will be moved, so if the title
```

Comment:
I guess that 'manualy' is a typo. Besides, the whole sentence seems quite long to me with many complements: I wonder if this makes it hard to read or not (as a French native speaker, I am a bit too biased to judge long sentences with complements of complements...). Would something like
```
However, titles that need to be moved are
at ``y=1.0``, so manual placing at 1.0 will be moved.  If the title is to
be placed at 1.0, it should be set to something near 1.0, like 1.001.
```
be OK too?

Reply:
Agreed.  Changed to be more verbose, and hopefully more clear.

```
Previously an axes title had to be moved manually if an xaxis overlapped
(usually when the xaxis was put on the top of the axes).  Now, the title
will be automatically moved above the xaxis and its decorators (including
the xlabel) if they are at the top.

If desired, the title can still be placed manually.  There is a slight kludge;
the algorithm checks if the y-position of the title is 1.0 (the default),
and moves if it is.  If the user places the title in the default location
(i.e. ``ax.title.set_position(0.5, 1.0)``), the title will still be moved
above the x-axis.  If the user wants to avoid this, they can
specify a number that is close (i.e. ``ax.title.set_position(0.5, 1.01)``) 
and the title will not be moved via this algorithm.
```